gapis/api/templates/vulkan_gfx_api_extras.tmpl

```diff
@@ -393,6 +397,10 @@ bool Vulkan::replayCreateVkDeviceImpl(Stack* stack, size_val physicalDevice,
 	        xcbInfo.window = info->window;
 	      }
 	      break;
+      case gapir::SurfaceType::Ggp:
+        createInfo = &ggpInfo;
+        ggpInfo.sType = VK_STRUCTURE_TYPE_STREAM_DESCRIPTOR_SURFACE_CREATE_INFO_GGP;
+        ggpInfo.streamDescriptor = *(const uint32_t*)data;
```

Comment:
Missing break - by chance it fallsthrough to the `default` with a single `break`.